src/neo4j-arc/cypher-language-support/cypher-editor/CypherEditor.tsx

```diff
@@ -385,11 +387,21 @@ export class CypherEditor extends React.Component<
       KeyMod.CtrlCmd | KeyCode.US_DOT,
       this.props.onDisplayHelpKeys
     )
-    this.editor.addCommand(
-      KeyCode.Escape,
-      this.props.toggleFullscreen,
-      '!suggestWidgetVisible && !findWidgetVisible'
-    )
+
+    function keys<T>(object: T) {
```

Comment:
Perhaps move this to the "objectUtils" file?

Reply:
Good call, missed that file